base/LocalIO.cpp

```diff
@@ -296,7 +296,7 @@ inline U8 Souliss_DigInHold_Helper(U8 pin, U8 pin_value, U8 value, U8 value_hold
 
 		return MaCaco_NODATACHANGED;
 	}
-	else if(pin_value && (abs(millis()-souliss_time) > holdtime) && (InPin[pin]==PINSET))
```

Comment:
All these changes come from https://github.com/souliss/souliss/pull/328 they should disappear from the diff once that is merged.